omero/sysadmins/server-backup-and-restore.rst

```diff
@@ -63,13 +63,14 @@ OMERO.server has three main backup sources:
     (assumed to be :file:`/OMERO`)
 3.  OMERO.server configuration
 
-.. warning:: You must back up *(1)* and *(2)* regularly.
+.. warning:: You must back up *(1)* and *(2)* frequently.
 
-Regular backups taken while the server is still running are usually
+Frequent backups taken while the server is still running are usually
 sufficient but you should be aware that they may not be consistent
 snapshots. The **safest** course of action is to perform
 backups during server downtime when possible, especially if you think you
-may need the backup.
+may need the backup. An alternative would be to restart the server so as
+to be in :ref:`read-only` mode for the duration of the backup process.
```

Comment:
If we're recomending read-only for backups I think we should have better docs than the developer page.

Reply:
We're not *recommending* it. :smiley:

Reply:
I'm concerned we're mentioning it in our sysadmin docs to people who may not have much knowledge of OMERO, but if they follow it up there really isn't much info on read-only.